drivers/SmartThings/zigbee-smoke-detector/profiles/gas-lifetime-selfcheck-aqara.yml

```diff
@@ -0,0 +1,28 @@
+name: gas-lifetime-selfcheck-aqara.yml
+components:
+- id: main
+  capabilities:
+  - id: gasDetector
+    version: 1
+  - id: refresh
+    version: 1
+  - id: audioMute
+    version: 1
+  - id: stse.selfCheck
+    version: 1
+  - id: stse.lifeTimeReport
+    version: 1
+  - id: stse.sensitivityAdjustment
+    version: 1
+    config:
+     values:
+       - key: "sensitivityAdjustment.value"
+         enabledValues:
+           - 'Low'
+           - 'High'
+       - key: "{{enumCommands}}"
+         enabledValues:
+           - 'Low'
+           - 'High'
+  categories:
+  - name: SmokeDetector
```

Comment:
Hi, @Hongming6 

Please change profile to 
```
name: gas-lifetime-selfcheck-aqara.yml
components:
- id: main
  capabilities:
  - id: gasDetector
    version: 1
  - id: refresh
    version: 1
  - id: audioMute
    version: 1
  - id: stse.selfCheck
    version: 1
  - id: stse.lifeTimeReport
    version: 1
  - id: stse.sensitivityAdjustment
    version: 1
  categories:
  - name: SmokeDetector
metadata:
  mnmn: SmartThingsCommunity
  vid: 407e7b4f-4e9c-3cb3-9849-bd0d57f9b40a
```

And attach a screenshot if possible.

Reply:
Thx,modified.
![Uploading 20828905726.JPG…]()